soc/riscv/riscv-privileged/Kconfig

```
@@ -28,6 +28,11 @@ config RISCV_HAS_CLIC
 	help
 	  Does the SOC provide support for a Core-Local Interrupt Controller (CLIC).
 
+config RISCV_CUSTOM_INTERRUPT_CONTROLLER
+	bool "Does the SOC provide support for a custom Interrupt Controller (IC)"
+	help
+	  Does the SOC provide support for a non-standard, custom Interrupt Controller (IC).
```

```suggestion
	bool "This option signifies that the RISCV CPU is connected to a custom interrupt controller"
	help
	  When this option is selected, the architecture interrupt functions...
```
Copy from the ARM help.